docs/Configuration.md

```diff
@@ -176,9 +176,9 @@ Mapper.Initialize(cfg => {
 });
 ```
 
-## Global property/field filtering
+## Global property/field/constructor filtering
 
-By default, AutoMapper tries to map every public property/field. You can filter out properties/fields with the property/field filters:
+By default, AutoMapper tries to map every public property/field, and all non-static declared constructors. You can filter out properties/fields/constructors with the property/field/constructor filters:
```

Comment:
I think the wording is strange, AM doesn't map constructors, it uses them to construct destination objects. Also there is a special page for constructors, so the content should be there, not here.

Reply:
I'll move it, I agree.